ci-operator/step-registry/azure/provision/custom-role/azure-provision-custom-role-commands.sh

```diff
@@ -319,11 +320,8 @@ ${required_permissions}
 """
     fi
 
-    #optional permissions for creating a private storage endpoint for internal image registry
-    registry_conf="${SHARED_DIR}/manifest_image_registry-config.yml"
-    if [[ -f "${registry_conf}" ]]; then
-        registry_type=$(yq-go r "${registry_conf}" 'spec.storage.azure.networkAccess.type')
-        if [[ "${registry_type}" == "Internal" ]]; then
```

Comment:
It's better to keep this condition as well, I remembered that internal-registry feature can also be enabled on private cluster, but not only on fullyprivate cluster (we just select fullyprivate job in CI currently)

Reply:
> One more questions: are following permissions all required for azure file csi driver in fullyprivate cluster? Do you think that doc need to be updated to reflect this?

I think so these permissions also needed for azure file csi driver in fullyprivate cluster.

Reply:
Updated and retest result looks good, @jinyunma could you help take a look again when you get a chance? Thank you!